lib/azure/armrest/storage_account_service.rb

```diff
@@ -66,6 +73,16 @@ def list(group = nil)
         end
       end
 
+      # List all storage accounts for the current subscription. This does not
+      # include storage account key information.
+      #
+      def list_subscription
```

Comment:
Normally we have pattern `list_###s` to list something in plural. Can we somehow rename this method to reduce misleading? Something like `list_for_current_subscription`, or `list_for_subscription` but I would expect a subscription to be in the argument list.